### ISSUE TITLE
Unable to Run Fusion++ on Windows Server 2016

### ISSUE THREAD

**Issue by elzik** [awaescher/Fusion#38]
I can run Fusion++ on my own Windows 10 development box but when I try and do the same on a Windows Server 2016 box (via a Chocolatey install or just by copying the binaries) I get the following message:

![image](https://user-images.githubusercontent.com/23397871/176680547-a66ed373-ab59-46bc-946f-1837df513122.png)

Clicking yes leads me to a download page for .NET 5. I have installed the x64, x86 and the hosting packages but still, on running the executable, I get the same prompt to install .NET 5.

These are the currently installed runtimes:
```
PS C:\Users\automation> dotnet --list-runtimes
Microsoft.AspNetCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.AspNetCore.App 6.0.4 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.NETCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 6.0.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
```

**Comment by awaescher** [comment]
Should be fixed with a self-contained build with 1.50, see #30

**Comment by RussKie** [comment]
Install WindowsDesktop runtime?
________________________________
From: elzik ***@***.***>
Sent: Thursday, 30 June 2022 10:54 PM
To: awaescher/Fusion ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [awaescher/Fusion] Unable to Run Fusion++ on Windows Server 2016 (Issue #38)


I can run Fusion++ on my own Windows 10 development box but when I try and do the same on a Windows Server 2016 box (via a Chocolatey install or just by copying the binaries) I get the following message:

[image]<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F23397871%2F176680547-a66ed373-ab59-46bc-946f-1837df513122.png&data=05%7C01%7Cigor.velikorossov%40microsoft.com%7Ce44090b383e645b927e808da5a97bbcd%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637921904999961616%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=2SWbgEP8j%2BEAUdosGU7BT9nRh%2BQ0Yv2hrSg%2Bc3udUS4%3D&reserved=0>

Clicking yes leads me to a download page for .NET 5. I have installed the x64, x86 and the hosting packages but still, on running the executable, I get the same prompt to install .NET 5.

These are the currently installed runtimes:

PS C:\Users\automation> dotnet --list-runtimes
Microsoft.AspNetCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.AspNetCore.App 6.0.4 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
Microsoft.NETCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 6.0.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
